core/core-api/src/main/kotlin/org/eclipse/hara/ddiclient/core/api/HaraClientData.kt

```diff
@@ -13,6 +13,18 @@ package org.eclipse.hara.ddiclient.core.api
 import java.net.MalformedURLException
 import java.net.URL
 
+/**
+ * Configuration data of hara client.
+ * @param tenant it is the device owner and provides
```

Comment:
@param tenant the name of the tenant (target group) on the Update Server to connect to.